nmmo/lib/spawn.py

```diff
@@ -127,12 +127,13 @@ def get_team_spawn_positions(config, num_teams):
   teams_per_sides = (num_teams + 3) // 4 # 1-4 -> 1, 5-8 -> 2, etc.
 
   sides = get_edge_tiles(config)
-  assert len(sides[0]) > 4*teams_per_sides, 'Map too small for teams'
+  assert len(sides[0]) >= 4*teams_per_sides, 'Map too small for teams'
 
   team_spawn_positions = []
   for side in sides:
     for i in range(teams_per_sides):
       idx = int(len(side)*(i+1)/(teams_per_sides + 1))
       team_spawn_positions.append(side[idx])
 
+  np.random.shuffle(team_spawn_positions)
```

Comment:
I deleted the random.shuffle, so that "left team" and "right team" references can be used for the team task

Reply:
without shuffling, aren't we always spawning agents in the same place during training?